src/accessibility/utils/data-processing.js

```diff
@@ -973,17 +974,80 @@ export async function sendRunImportMessage(
   });
 }
 
+/**
+ * Retrieves code path information saved in S3 bucket
+ * @param {Object} site - The site object
+ * @param {string} opportunityType - Opportunity type for logging
+ * @param {Object} context - The context object containing log, s3Client, env
+ * @returns {Promise<Object|null>} Object containing codeBucket and codePath, or null if should skip
+ */
+async function getCodeInfo(site, opportunityType, context) {
+  const { log, s3Client, env } = context;
+  const siteId = site.getId();
+  const deliveryType = site.getDeliveryType();
+  const codeConfig = site.getCode();
+
+  // For aem_edge delivery type, proceed without codeConfig
+  if (!codeConfig) {
+    if (deliveryType === 'aem_edge') {
+      return {
+        codeBucket: env.S3_IMPORTER_BUCKET_NAME,
+        codePath: '',
```

Comment:
any specific reason for the same ?

Reply:
Even if codeConfig is not defined for EDS, we would like to generate a git patch